### ISSUE TITLE
Greater/less than + other relational operators too strict on union types

### ISSUE THREAD

**Issue by RyanCavanaugh** [Microsoft/TypeScript#11573]
master branch as of 10/12/2016

```ts
let lhs: 10 | 20 = <any>10;
let rhs: 15 | 25 = <any>15;
if (lhs > rhs) { // Error, can't compare 10|20 to 15|25

}
```
Actual behavior:Error, can't compare 10 | 20 to 15 | 25 
Expected: No error

**Comment by RyanCavanaugh** [comment]
Duplicate #10119
